docs/website/root/manual/developer-docs/nodes/mithril-client.md

```diff
@@ -296,20 +300,35 @@ Here is a list of the available parameters:
 | `network` | - | - | `NETWORK` | Cardano network | - | `testnet` or `mainnet` or `devnet` | :heavy_check_mark: |
 | `aggregator_endpoint` | `--aggregator-endpoint` | - | `AGGREGATOR_ENDPOINT` | Aggregator node endpoint | - | `https://aggregator.pre-release-preview.api.mithril.network/aggregator` | :heavy_check_mark: |
 | `genesis_verification_key` | - | - | `GENESIS_VERIFICATION_KEY` | Genesis verification key | - | - | :heavy_check_mark: |
-| `json_output` | `--json` | `-j` | - | Enable JSON output | no | - | - |
+| `log_format_json` | `--log-format-json` | - | - | Enable JSON output for logs | no | - | - |
+| `log_output` | `--log-output` | - | - | Redirect the logs to a file | no | `./mithril-client.log` | - |
```

Comment:
It looks like the `-o` shortcut is missing in this doc.

Also (same comment for all the other occurrences below):
```suggestion
| `log_format_json` | `--log-format-json` | - | - | Enable JSON output for logs | - | - | - |
| `log_output` | `--log-output` | - | - | Redirect the logs to a file | - | `./mithril-client.log` | - |
```